torchrl/envs/transforms/transforms.py

```diff
@@ -312,9 +321,16 @@ def __init__(
                 # we don't use isinstance as some transforms may be subclassed from
                 # Compose but with other features that we don't want to loose.
                 transform = [transform]
+            else:
+                for t in transform:
+                    t.reset_parent()
```

Comment:
same thing: does this modify the original `TransformedEnv` in any way?

Reply:
I was wondering the same thing - Since a new `TransformedEnv` object is created in `transform.parent` we would need to make copies of all the transforms added to this new object in order to keep the original `_parent` attributes pointing to the correct (original) 'TransformedEnv'. Does that make sense to do here?